fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageSession.java

```diff
@@ -253,9 +254,16 @@ public RdfStream getTriples(final String identifier, final Instant version)
 
     @Override
     public InputStream getBinaryContent(final String identifier, final Instant version)
```

Comment:
Should this not take a transaction? What if the binary is created and then `GET`ted inside a single long-running transaction?

Reply:
The class this method is in is effectively the transaction at the persistence layer, so it should be resolving to the state of the long running transaction that the GET is taking place in

Reply:
This went 90% over my head, but I think I got enough to be okay.